page-objects/pages/TASKPage.js

```diff
@@ -23,6 +23,7 @@ const submissionRow = `${submissionsTable} tbody tr.userinfo`;
 const openTasksTab = "//span[@data-testid = 'openTasks']";
 const completedTasksTab = "//span[@data-testid = 'closedTasks']";
```

Comment:
Closed and finished tasks sound very similar. Where's the difference? Does it depend on the user that is logged in?